### PR TITLE
Remove some Jelastic Domains from PSL due to their end of service (Mods #1023 #1063 #1092 #1095 #1151)

### PR DESCRIPTION
**Pull request by ihorman**
* [x] Description of Organization
* [x] Reason for PSL Removal
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration.

Description of Organization
====

Jelastic is a Multi-Cloud PaaS (Platform as a Service) with decentralized network of independent cloud hosting service providers located across the Globe.

Organization Website: https://jelastic.com

Reason for PSL Removal
====

The domains cloudswitches.com and leviracloud.eu are removed from PSL due to end of services, that were provided by the partners. 
The owner of shared domain elastyco.com has requested to remove his domain from PSL.

DNS Verification via dig
=======
```
dig txt +short _psl.elastyco.com                                                                                                                        ✔  10061  
"https://github.com/publicsuffix/list/pull/1188"
```
Other domains are eliminated by the partners.

make test
=========

Tested OK.

### CODE REVIEW

**Comment by dnsguru** [comment]
Mods #1023 #1063 #1092 #1095 #1151

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12216,6 +12218,7 @@ us.reclaim.cloud
 ch.trendhosting.cloud
 de.trendhosting.cloud
 jele.club
+amscompute.com
```
Comment (dnsguru) [inline public_suffix_list.dat:12221]: domain expires in 10 months
no _psl txt record to validate

**Comment by dnsguru** [comment]
ALL ADDED NAMES MUST HAVE >2 YEARS AND THEIR DNS _PSL TXT RECORDS FOR VALIDATION

AMSCOMPUTE.COM (perhaps others, stopped there) DID NOT.

MAKE SURE **ALL** SUBMITTED NAMES COMPLY WITH REQUIREMENTS FOR INCLUSION.

REVIEW YOUR LIST, CORRECT THE ISSUES WITH REGISTRATION TERM AND ADD _PSL TXT RECORDS

File: public_suffix_list.dat (hunk 1 above)
Comment (ihorman) [reply public_suffix_list.dat:12221]: @dnsguru  please accept my apologies, my colleague  accidentally merged new domain to a wrong branch in our PSL fork,
asked him to revert changes and arrange another pull request for new domain once everything will be ready for that
thank you

**Comment by dnsguru** [comment]
Closing due to non-response >30 days

**Pull request closed without merging**